packages/web-runtime/src/extensions.ts

```diff
@@ -2,6 +2,7 @@ import { computed, markRaw } from 'vue'
 import { preferencesPanelExtensionPoint, progressBarExtensionPoint } from './extensionPoints'
 import AppTokens from './components/Account/AppTokens.vue'
 import { CustomComponentExtension, LoadingIndicator } from '@opencloud-eu/web-pkg'
+import CalDavUrl from './components/Account/CalDavUrl.vue'
```

Comment:
Not 100 percent sure about importing this, I’d rather have an extension point not be dependent on a component, maybe you find a better place for the constant.